src/OpenLoco/Map/QuarterTile.h

```diff
@@ -1,3 +1,4 @@
+#pragma once
```

Comment:
Can you add one blank line after each pragma to match other headers?

Reply:
I now have added the black line after the new pragmas